ingest.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "Starting ingest..."
+source ./venv/Scripts/activate
+python ingest.py
```

Comment:
```suggestion
python3 ingest.py
```
This would be preferred everywhere I think.

Reply:
My be it is a good idea to pass venv and python paths as optional params as in Stable Diffusion Web UI for example.